presto-native-execution/presto_cpp/main/types/tests/PlanConverterTest.cpp

```diff
@@ -99,13 +98,16 @@ class PlanConverterTest : public ::testing::Test {
  protected:
   static void SetUpTestCase() {
     memory::MemoryManager::testingSetInstance({});
+    registerPrestoToVeloxConnector(
+        std::make_unique<HivePrestoToVeloxConnector>("hive"));
+    registerPrestoToVeloxConnector(
+        std::make_unique<HivePrestoToVeloxConnector>("hive-plus"));
```

Comment:
Is hive-plus even used?

Reply:
It is used in `ScanAggCustomConnectorId.json`